### ISSUE TITLE
GGMAX Enhancement - Text field for display on Huds

### ISSUE THREAD

**Issue by Necrym59** [TheGameCreators/GameGuruRepo#5645]
On HUD's we have _prompt:main_ as a display variable to use in the main hud for display of 'prompt' text in-game. Can we make another called _hudtext:main_  for use with new a 'hudtext(str)' Lua command for other text sometimes required from behaviors (and there are quite a few) that then can be passed through to the user hud variable so the user can position/size however they want this text to be displayed.

**Comment by LeeBamberTGC** [comment]
@Necrym59 So the idea is that a HUDTEXT:MAIN text gadget can be dragged into the screen editor of a HUD screen, similar to this:
![image](https://github.com/user-attachments/assets/ec30aa39-fc29-47eb-9af6-83cead172e6c)

How would the user then use this additional gadget that is any different from simply creating a custom name and then detecting this name and changing the position of the element within the HUD screen?  

Are you perhaps looking to take the size/font/color from the gadget element and using it when "Text(x,y,size,txt)" command is called?  Or do you envisage new LUA commands such as HUDTEXT(str) and what would this do?

**Comment by Necrym59** [comment]
Yep thats it, Just somthing like HUDTEXT(str) similar to the way prompt works is fine no need to set the color etc. its just so other text in behaviors can be passed to HUDTEXT(str) and displayed in huds as required.

**Comment by Necrym59** [comment]
Actuall negate this one as this one covers this idea
https://github.com/TheGameCreators/GameGuruRepo/issues/6032
